.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "3rdparty/GSL_LITE"]
 	path = 3rdparty/GSL_LITE
 	url = https://github.com/martinmoene/gsl-lite.git
+[submodule "3rdparty/lexertl14"]
```

Comment:
F'in Github won't allow me to comment above, but you still have `3rdparty/lexertl` in this file.

Reply:
I'll remove it, but it shouldn't matter (submodules are less than ideal)